packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/virtualMachine/dedicatedCloud-datacenter-virtualMachine.service.js

```diff
@@ -0,0 +1,15 @@
+export default class {
+  /* @ngInject */
+  constructor(OvhHttp) {
+    this.OvhHttp = OvhHttp;
+  }
```

Comment:
Prefer using $http instead, OvhHttp is deprecated

```suggestion
  constructor($http) {
    this.$http = $http;
  }
```